index.html

```diff
@@ -285,19 +283,22 @@ <h2 class="header-text">About <br>Myself</h2>
       </div>
     </div>
   </section>
-  <section class="contact-form">
-    <div class="contact-me">
-      <h2>Contact me</h2>
-      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-    </div>
-    <form action="https://formspree.io/f/myyawnzb"  method="post">
-      <input class="input-1" type="text" placeholder="Yeremias"  maxlenght="30" required>
-      <input class="input-2" type="email" name="email" placeholder="natajayanj14@gmail.com" maxlength="30" required>
-      <textarea class="input-3" name="textarea" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
-      <button id="btn">Get in touch</button>
-      <img class="footer-shape" src="./images/Shape.svg" alt="">
-    </form>
-  </section>
+  <div class="contact-container">
+    <section class="contact-form">
+      <div class="contact-me">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      </div>
+      <form action="https://formspree.io/f/myyawnzb" method="post">
+        <input class="input-1" type="text" placeholder="Yeremias" maxlenght="30" required>
+        <input class="input-2" type="email" name="email" placeholder="natajayanj14@gmail.com" maxlength="30" required>
+        <textarea class="input-3" name="textarea" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here"></textarea>
+        <button id="btn">Get in touch</button>
+        <img class="footer-shape" src="./images/Shape.svg" alt="">
+      </form>
+    </section>
```

Comment:
- [ ] kindly consider giving this button some padding-bottom so as to this section match the provided design 
- 
![pad](https://user-images.githubusercontent.com/60146030/220991360-2d0d61db-445a-4167-941e-641372f520e6.PNG)